addons/website/static/src/systray_items/edit_website.js

```diff
@@ -23,6 +23,9 @@ class EditWebsiteSystray extends Component {
         useEffect((snippetsLoaded) => {
             if (snippetsLoaded) {
                 this.state.isLoading = false;
+                // Once the sinppet is loaded, set the focus on the search input.
+                this.snippetSearchInput = document.querySelector('.o_snippet_search_filter_input');
+                this.snippetSearchInput.focus();
```

Comment:
As discussed on the task of this PR, this code should be in `web_editor` as we want the same behavior in `Email Marketing`, as the same way it was done for empty the input when switching tabs.

Reply:
OK, Agree, initially it was decided to do changes only for website, as we are going keep behavior same, web_editor make sense.